packages/graph/hash_graph/lib/graph/src/datastore/mod.rs

```diff
@@ -123,6 +123,11 @@ impl DatabaseConnectionInfo {
         &self.user
     }
 
+    #[must_use]
+    pub fn password(&self) -> &str {
```

Comment:
Let's just add a warning message and not worry too much about pub/private, etc. This struct is used by the calling lib so honestly it already has access to the password because it set it in the first place.

We're considering moving from SQLx anyway so perhaps another library handles this better 🙃